controllers/admin/AdminLegacyLayoutController.php

```diff
@@ -100,6 +100,14 @@ public function __construct(
         $this->useRegularH1Structure = $useRegularH1Structure;
     }
 
+    /**
+     * This helps avoiding handling legacy processes when in Symfony Controllers.
+     * Otherwise when using POST action to render form you sometimes get an exception.
+     */
+    public function initProcess()
```

Comment:
Hoho this one could introduce dangerous side-effects because it will disable this for _every migrated page_ (ping @atomiix who works on the Legacy Layout migration).

Reply:
OK, I checked the code of `AdminController::initProcess()` and it seems nothing in there is required for migrated pages. So this should not create issues (in theory 😅 )

Reply:
Yeah this place is my biggest concern since it's such a fundamental function. But so far couldn't figure out anything that could go wrong.